doc/develop/toolchains/crosstool_ng.rst

```diff
@@ -25,6 +25,8 @@ You can build toolchains from source code using crosstool-NG.
 
    - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``xtools``.
    - Set :envvar:`XTOOLS_TOOLCHAIN_PATH` to the toolchain build directory.
+   - Set :envvar:`XTOOLS_CROSS_COMPILE_TARGET` to the target triplet of the toolchain,
+     e.g. ``arm-zephyr-eabi``.
```

Comment:
maybe a better option is to add a note, that if you're customizing the crosstool built toolchain (compared to Zephyr setup) then follow the cross compile toolchain setup.

Reply:
Yup, and that all works now. The second patch in this series allows a cross-compile variant toolchain to select TLS if it is supported. Thanks much for your help.